src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,39 @@
-export const Tabs = () => {};
+export const Tabs = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const findTab = tabs.find(tab => selectedTabId === tab.id) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              key={tab.id}
+              className={selectedTabId === tab.id ? 'is-active' : ''}
```

Comment:
```suggestion
              className={findTab.id === tab.id && 'is-active'}
```

it should help to pass the tests, also in this case better use `classnames` library

Reply:
It helped, thanks for advice!